src/ui/wxWidgets/PWSgrid.cpp

```diff
@@ -78,6 +79,18 @@ PWSGrid::PWSGrid(wxWindow* parent, PWScore &core,
 {
   Init();
   Create(parent, id, pos, size, style);
+  
+  // Handler for double click events on column header separator
+  auto *header = wxGrid::GetGridColHeader();
+  
+  if (header) {
+    header->Bind(
```

Comment:
Is it necessary to unbind the handler (e.g. in the dtor) to free up any system resources?

Reply:
To be honest, I'm not sure. If it is necessary, than how can this be done in case of lambda's?
In case of doubt I can choose the classical implementation for event handler's.

Reply:
Checked the [docs](http://docs.wxwidgets.org/trunk/overview_events.html#overview_events_bind) - there's no mention of the requirement to unbind.

Reply:
Checked for memory leaks with and without this patch, but haven't found anything suspicious pointing to the application.